be/src/util/trace_metrics.cpp

```diff
@@ -52,8 +52,6 @@ static InternMap* g_intern_map;
 
 const char* TraceMetrics::InternName(const string& name) {
     DCHECK(std::all_of(name.begin(), name.end(), [](char c) { return isprint(c); })) << "not printable: " << name;
-
-    debug::ScopedLeakCheckDisabler no_leakcheck;
```

Comment:
This is used to disable Leak check in its scope. So is should not be removed

Reply:
ok, got it. reverted this place.